nshmdb/nshmdb.py

```diff
@@ -21,20 +21,67 @@
 >>> db.get_rupture_faults(0) # Should return two faults in this rupture.
 """
 
+import collections
 import dataclasses
 import importlib.resources
 import sqlite3
 from pathlib import Path
 from sqlite3 import Connection
+from typing import Optional
 
+import duckdb
 import numpy as np
-import qcore.coordinates
+from qcore import coordinates
+from source_modelling.sources import Fault, Plane
 
-from nshmdb import fault
-from nshmdb.fault import Fault
+from nshmdb import query
 
 
 @dataclasses.dataclass
+class Rupture:
```

Comment:
Might be worth including the fault metdata as well? I.e. `FaultInfo` dict

Reply:
you can get that from a separate call to get fault info so I don't think it's super necessary.